recipes/quickfast/all/conanfile.py

```diff
@@ -16,7 +17,7 @@ class QuickfastConan(ConanFile):
                "shared": [True, False]}
     default_options = {"fPIC": True,
                        "shared": False}
-    requires = ["boost/1.73.0", "xerces-c/3.2.3"]
+    requires = ["boost/1.69.0", "xerces-c/3.2.3"]
```

Comment:
If `1.73.0` compiles I think you can put it back (no one will use such an old boost)

Reply:
FIxed by 73149ab4